victoriametrics.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: victoriametrics
```

Comment:
If the package only ships a subpackages and nothing else, can you call this package vicrotiametrics-vmagent directly?

Reply:
@developer-guy plans to add several more subpackages in the near future (that are separate apps in the same monorepo)